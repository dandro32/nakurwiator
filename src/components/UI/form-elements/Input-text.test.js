import React from 'react';
import InputText from './Input-text';
import ShallowRenderer from 'react-test-renderer/shallow';

describe('InputText test', () => {
  let shallowRendererResult;
  const mockFunc = jest.fn();
  beforeEach(() => {
    const shallowRenderer = new ShallowRenderer();
    shallowRenderer.render(<InputText label="label" value="value" required={true} onChange={mockFunc} name='name' />);
    shallowRendererResult = shallowRenderer.getRenderOutput();
  });

  it('should check is render is properly output', () => {
    expect(shallowRendererResult).toMatchSnapshot();
  });

});