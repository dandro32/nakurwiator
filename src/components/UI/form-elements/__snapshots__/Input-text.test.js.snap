// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputText test should check is render is properly output 1`] = `
<div
  className="form-group"
>
  <label>
    label
  </label>
  <input
    className="form-control"
    name="name"
    onChange={[Function]}
    required={true}
    type="text"
    value="value"
  />
  )
</div>
`;
